mender/templates/gui-svc.yaml

```diff
@@ -31,7 +31,7 @@ spec:
   ports:
   - port: {{ .Values.gui.service.port }}
     protocol: TCP
-    targetPort: 80
+    targetPort: {{ .Values.gui.httpPort }}
```

Comment:
Unfortunately the `gui` container is not currently supporting non-root users. Please remove the modifications here for now.